### PR TITLE
New neurolincs instance

### PR DESCRIPTION
**Pull request by philerooski**
Replaces #80

### CODE REVIEW

File: config/prod/neurolincs-actual.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v4.yaml
```
Comment (thomasyu888) [inline config/prod/neurolincs-actual.yaml]: Use new template

Comment (philerooski) [reply config/prod/neurolincs-actual.yaml]: Is v4 no longer supported?

Comment (thomasyu888) [reply config/prod/neurolincs-actual.yaml]: I don't think its not supported, but you always want to use the newer versions as they become available.

Comment (philerooski) [reply config/prod/neurolincs-actual.yaml]: I understand, but v4 works for my needs. If you no longer want people to open PRs using the v4 or v3 templates you should deprecate and remove them from the templates directory. I'll be happy to reopen this PR using v5 if you decide to do so.

Comment (zaro0508) [reply config/prod/neurolincs-actual.yaml]: both of you are correct.  I forgot to deprecate managed-ec2-v4.yaml.  It is deprecated now. https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/templates/managed-ec2-v4.yaml

**Comment by philerooski** [comment]
@zaro0508 @thomasyu888 changes made

**Pull request merged**